lib/backup/s3remote/s3.go

```diff
@@ -97,6 +99,11 @@ func (fs *FS) Init() error {
 	configOpts := []func(*config.LoadOptions) error{
 		config.WithSharedConfigProfile(fs.ProfileName),
 		config.WithDefaultRegion("us-east-1"),
+		config.WithRetryer(func() aws.Retryer {
```

Comment:
default retry settings:
1) 3 retries - should we raise it ?
2) 20 seconds back-off.

Reply:
1. Increased amount of retries in https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6747/commits/b538a00712ecaa3eb78be399a2fc381bbd9844f2.
With 3 minutes limit exponential backoff should perform up to 9 retries.

2. `retry.NewExponentialJitterBackoff` creates a backoff policy with exponential delay, so it will override 20 seconds default.